Source/Mac/NonModalAlertWindowController.swift

```diff
@@ -58,13 +54,13 @@ class NonModalAlertWindowController: NSWindowController {
         newFrame.origin.x += oldFrame.size.width - newFrame.size.width
         confirmButton.frame = newFrame
 
-        if let cancelButtonTitle {
+        if let cancelButtonTitle = cancelButtonTitle {
             cancelButton.title = cancelButtonTitle
             cancelButton.sizeToFit()
-            var adjustFrame = confirmButton.frame
```

Comment:
It looks like my bad. I guess McBopomodo has the same issue.

Reply:
No worries. This has been fixed in https://github.com/openvanilla/McBopomofo/pull/468 earlier, and that's why this PR syncs the code there.